external/llvm-project/mlir/include/mlir/Conversion/GPUToROCDL/GPUToROCDLPass.h

```diff
@@ -42,7 +42,6 @@ void configureGpuToROCDLConversionLegality(ConversionTarget &target);
 /// is configurable.
 std::unique_ptr<OperationPass<gpu::GPUModuleOp>>
 createLowerGpuOpsToROCDLOpsPass(
-    const std::string &chipset = "gfx900",
```

Comment:
For backwards compatibity/other people using the pass/..., keep this, but use a default value that's something like `target` or `infer`

Reply:
Added "infer" option